app/scrapers/bing.py

```diff
@@ -8,17 +6,21 @@ class Bing:
     def __init__(self):
         pass
 
-    def get_page(self,query):
+
+    def get_page(self, query):
         """
         Fetches search response from bing.com
         returns : result page in html
         """
         header = {
-            'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_8_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/27.0.1453.116 Safari/537.36'}
+            'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_8_2) \
```

Comment:
Backslashes are undesirable because whitespace to the right of the backslash breaks the program yet is invisible to the reader.  See PEP8.  Use parentheses instead.
```python
    long_string = ('                  '
                   '                  ')
```